frontend/src/app/components/channel/large/large.controller.ts

```diff
@@ -10,4 +12,22 @@ export class LargeChannelController extends SingleUserChannelController {
     }
   }
 
+  onDragUserFrom(data: IDraggedData) {
+    if (data.destinationType === 'search') {
+      return;
+    }
+    this.users.some((user: IUser, index: any) => {
+      if (user.login !== data.user.login) {
+        return false;
+      }
+      if (data.destinationType === 'disable') {
+        this.disableUser(user);
+      } else {
+        this.users.splice(index, 1);
+      }
+      return true;
+    });
+    this.usersByLogin = {};
```

Comment:
delete from object instead of rebuilding it. performance considerations.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete